### DISCUSSION TITLE
Giex G02 - Zigbee2Mqtt

### DISCUSSION THREAD

**Discussion by reclus23** [indomus/forum#3046] [HUB: Home Assistant]
ciao ho installato, dopo aver letto su indomus che veniva riconosciuto senza problemi, tramite zigbee2mqtt, il sisttema di irrigazione della Giex (gx02).
L'apparecchio è stato correttamente riconosciuto  ed alla pagina dei valori ho tutto.
Il problema che riscontro è che appena clicco su "state" per accenderlo, tempo 1 secondo e si spegne. Da quello che vedo, materialmente l'irrigatore non si avvia nemmeno.
Ho messo le varie voci nella plancia di HA (ed i valori come per es, batteria, linkquality (180)) si vedono correttamente. Come clicco su state per farlo partire, nulla...il pulsante si attiva e si rispegne. Materialmente l'irrigatore nemmeno parte.
Se clicco state non dalla plancia ma dalla pagina "valori" di zigbee2mqtt, il pulsante state rimane su on ma l'irrigatore non si avvia...
(ho HA su VM ed uso skyconnect come bridge).
Dove sto sbagliando?
grazie

**Comment by quickgold1** [comment]
Ciao.
Stò impazzendo anch'io con questo prodotto.
Una volta configurato(perlomeno il mio) funziona, si accende e si spegne , anche come programmazione.
Ma non si sà come mai, dopo un pò di tempo, perde la connessione e non funziona più.
Poi premo di nuovo il pulsante, si riconnette e funziona, ma poi, di nuovo perde la connessione.
E per una elettrovalvola che deve assicurare l'innaffio, non è il massimo.
Spero vivamente che con un aggiornamento futuro, possa risolvere il problema.

**Reply by reclus23** [reply]
anche a me la stessa cosa. appena configurato funzionava. dopo qualche ora ho riprovato e niente.....
La cosa strana è che prima che venisse implementato completamente su zigbee2mqtt funzionava bene (accensione e spegnimento).
Con l'implementazione e riconoscimento automatico, riscontro questi problemi...

**Comment by quickgold1** [comment]
Ho letto sui forum internazionali,che sarebbe un problema di perdita di
connessione.
Infatti se quando non funziona, premi il pulsante,si riconnette e per un
po' rimane connesso e puoi comandarlo. Poi dopo qualche ora lo perde.
Speriamo si risolva con un aggiornamento.

Il sab 20 apr 2024, 21:41 reclus23 ***@***.***> ha scritto:

**Reply by reclus23** [reply]
bella scocciatura....averlo e non poterlo utilizzare....speriamo siano veloci nell'aggiornamento...ma poi come si aggiornerà? (scusate la domanda banale)

**Comment by quickgold1** [comment]
Ieri Sera ho provato una configurazione.
Mettendo come target 600 sec.
Cycle interval 13800 sec e cycle num times 100.
E messo così,con un intervallo tra un innaffio e un altro più ravvicinato
per ora non mi ha perso la connessione.
Incrocio le dita.

Il dom 21 apr 2024, 10:41 reclus23 ***@***.***> ha scritto:

**Comment by quickgold1** [comment]
Aggiornamento.
Niente da fare,tiene la connessione per 1 giorno,1 giorno e mezzo e poi
perde la connessione.
Ho provato anche a mettere un ripetitore a 2 metri di distanza, ma niente.
Aspetto e spero soluzioni,ma mi sa' che questo aggeggio ritorna nella
scatola da dove è venuto.

Il dom 21 apr 2024, 22:57 Vincenzo Grazzani ***@***.***> ha
scritto:

**Reply by reclus23** [reply]
alla fine ho preso un altro irrigatore.. questo della giex lo metto in un'altra parte dove non mi serve con home assistant...
però è un peccato perchè ha molte funzioni ed è bello solido...
